llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -83,13 +88,6 @@ class VPBuilder {
   VPBasicBlock *getInsertBlock() const { return BB; }
```

Comment:
Would it be sufficient to retrieve the plan from the insert block?

Reply:
as in `getInsertBlock()->getParent()->getPlan()`

Reply:
I already tried that, but it is unreliable, and I get crashes due to the insertion point not being set.